src/sortStudents.ts

```diff
@@ -1,16 +1,82 @@
+// import { objectExpression } from "@babel/types";
+// import { SignatureDeclarationBase } from "typescript";
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
```

Comment:
```suggestion
  Name = 'name',
```
it will help you to join similar cases (you will be allow to use it as computed property)